en_US/getting-started/getting-started.md

```diff
@@ -4,17 +4,34 @@ Starting from download and installation, this document guides you through quickl
 
 ## Select EMQX Edition
 
+{% emqxce %}
 EMQX as a software is released in different editions.
 
 - [Enterprise](https://www.emqx.com/en/try?product=enterprise)
 - [Community](https://www.emqx.com/en/try?product=broker)
 
-EMQ as a company offers a fully managed MQTT cloud service [EMQX Cloud](https://www.emqx.com/en/try?product=cloud)
-which runs EMQX Enterprise edition.
+{% emqxce %}
+
+EMQ as a company offers a fully managed MQTT cloud service [EMQX Cloud](https://www.emqx.com/en/try?product=cloud) which runs EMQX Enterprise edition.
 You can customize the deployment to perfectly suit your business development plans and get started quickly.
 
 :::: tabs type:card
 
+{% endemqxce %}
+::: tab EMQX Community Edition
+The world's most scalable distributed MQTT broker with a high-performance real-time message processing engine, powering event streaming for IoT devices at a massive scale.
+
+- Fully open-sourced under APL 2.0
+- MQTT 3.1.1 and MQTT 5.0 spec
```

Comment:
Previously we mention handling _all_ MQTT clients, and we definitely have a doc for MQTT-SN protocol. Perhaps we should call out MQTT 3.1.0 and MQTT-SN versions here as well to indicate that everything is supported?

Reply:
yes